### PR TITLE
Event Music & Nuke Countdown Song

### PR DESCRIPTION
**Pull request by ike709**
## About the PR 

Modifies AdminSoundSystem to be a more generic system for playing sounds globally to players, with the only restrictions being if they've disabled the sound via cvar or (optionally) if they're not on the same grid.

Also as an example usage, adds [Countdown](https://www.youtube.com/watch?v=G2gVAPKlgqA) from the BeeStation OST to when the nuke is armed. It's our second most-popular song by views.

I eventually intend to add support to everything Bee has songs for, like xeno infestation and war ops.

**Screenshots**
Note: I had to do a fresh install of OBS so the resolution is wonky but you're just here for the audio anyways.

It starts when armed, stops when disarmed, and will start again if rearmed. Make sure you unmute audio:

https://user-images.githubusercontent.com/5714543/171702349-d0eb251c-2e1f-4786-ba66-18d7fba58a46.mp4

**Changelog**

:cl:
- add: A song now plays during the nuke countdown. It can be disabled with the Event Music audio setting.

### CODE REVIEW

**Comment by DogZeroX** [comment]
Also, the song ends 67 seconds before the nuke explodes, so we will have to lower the nuke time until it explodes or have an awkward period of silence before detonation.

**Comment by ike709** [comment]
It's timed for SS13's default of 90 seconds. Don't really know what to do about SS14 using 180 seconds.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Server/Nuke/NukeSystem.cs [hunk 1]
```diff
@@ -345,6 +347,8 @@ public void ArmBomb(EntityUid uid, NukeComponent? component = null)
             // todo: move it to announcements system
             SoundSystem.Play(Filter.Broadcast(), component.ArmSound.GetSound());
```
Comment (metalgearsloth) [inline Content.Server/Nuke/NukeSystem.cs:348]: This is also broadcast.

**Comment by wrexbe** [comment]
Are you still working on this?

**Comment by ike709** [comment]
Yes.

**Comment by ike709** [comment]
> Are you still working on this?

\> leave as draft for 2 days
\> asked if derelict
\> forensics has been waiting for a review for 10 days

>:^(

**Comment by mirrorcult** [comment]
thank you for making me realize i never submitted my review on forensics

**Comment by metalgearsloth** [comment]
Ideally we wouldn't have like 3 different content audio systems + engine but can fix that later.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**